apps/ergw_core/src/ergw_gtp_c_socket.erl

```diff
@@ -174,8 +174,8 @@ init({Name, #{ip := IP, vrf := VRF,
 	       v1_seq_no = 0,
 	       v2_seq_no = 0,
 	       pending = gb_trees:empty(),
-	       requests = ergw_cache:new(?T3 * 4, requests),
-	       responses = ergw_cache:new(?CACHE_TIMEOUT, responses),
```

Comment:
Looks like the `CACHE_TIMEOUT` macros is not used anymore in current module. Please delete this non use macros.